### DISCUSSION TITLE
Using AdminClient with Strimzy

### DISCUSSION THREAD

**Discussion by ArtoLord** [strimzi/strimzi-kafka-operator#8365] [Q&A] [answered]
Hi!

I want to use strimzi to deploy Kafka in K8s.
I need to create users, tops and ACLs in runtime. However, I ran into the fact that if I create users via Kafka Admin API, after a while the User entity operator removes them because they are not in K8s resources.

What is the best way to deal with this situation? Is it possible to use admin API with strimzi?

**Comment by scholzj** [comment, marked as answer]
If you want to manager users, their ACLs or topics your self, you should disable the User / Topic operators. You can do so by removing the `userOperator` / `topicOperator` (or the whole `entityOperator`) sections from the `Kafka` custom resource and they will not be used. Then it is just up to you to manage the things with the Kafka Admin API.

**Reply by ArtoLord** [reply]
How do I create an administrator user in this case?
I couldn't find a parameter to create admin ACLs and credentials in the Kafka resource.

**Reply by scholzj** [reply]
There are not any. You have to follow what you would do with any regular Kafka cluster. E.g. start with unauthenticated access first to set things up, use a super user in the authorizer configuration etc.

**Reply by ArtoLord** [reply]
Ok, thanks for the answer, I'll try this
